README.md

```diff
@@ -140,6 +140,55 @@ env:
   SET_PUBLISH_DATE: true
 ```
 
+## Multiple shows per repository
```

Comment:
I'm fine with leaving these instructions as a seperate part in the README as it is right now
If I can just ask you to add it to the table of contents

Reply:
solved by a4dbba703cab726bac4a6ca564322c93c4473bed